pandas/tests/arrays/test_timedeltas.py

```diff
@@ -51,6 +63,17 @@ def test_copy(self):
 
 
 class TestTimedeltaArray(object):
+    def test_np_sum(self):
+        # GH#25282
+        # TODO: result should be a Timedelta, not a np.timedelta64 object
+        vals = np.arange(5).view('m8[h]').astype('m8[ns]')
+        arr = TimedeltaArray(vals)
+        result = np.sum(arr)
+        assert result == vals.sum()
+
+        result = np.sum(pd.TimedeltaIndex(arr))
+        assert result == vals.sum()
```

Comment:
* Does that `TODO` refer to both tests or just the first `assert` ?
* Similar feeling about parameterization...parts of me would like to, but it's not as nice as others...

Reply:
> Does that `TODO` refer to both tests or just the first `assert` ?

Both, I think. Or at least the index and array class should be consistent. But instead of a TODO, I would maybe rather open an issue about it.

Reply:
will remove TODO and open an issue.

@gfyoung I don't think either of these cases merit parametrization, but I _did_ take your comments into consideration and give the matter real thought.